include/common_structs.h

```diff
@@ -106,7 +114,6 @@ typedef struct Npc {
     /* 0x0AA */ u8 renderMode;
     /* 0x0AB */ char unk_AB[661];
 } Npc; // size = 0x340
-
```

Comment:
Can we keep the line of space between these structs?

Reply:
Why is it there?